### DISCUSSION TITLE
livewire/update 404

### DISCUSSION THREAD

**Discussion by skyville0919** [livewire/livewire#8213] [1. Help]
All actions lead to 404 not found. 
Localhost seems to works fine but after uploading the app to aws ec2 with nginx all actions calling /update and showing 404 not found. I have tried solutions on other discussions I have read but no luck on any of it.

**Comment by joshhanley** [comment]
@skyville0919 not sure how to help other than to say you will need to track down where in the codebase the 404 is originating from. If you can work that out, we might be able to guide on how to fix the problem.
